src/util/env.js

```diff
@@ -11,11 +11,13 @@ export const inBrowser =
 // detect devtools
 export const devtools = inBrowser && window.__VUE_DEVTOOLS_GLOBAL_HOOK__
 
-// UA sniffing for working around browser-specific quirks
+// UA sniffing for working around browser-specific quirks, distinguish UIWebView from WKWebView by indexedDB
 const UA = inBrowser && window.navigator.userAgent.toLowerCase()
 export const isIE9 = UA && UA.indexOf('msie 9.0') > 0
 export const isAndroid = UA && UA.indexOf('android') > 0
 export const isIos = UA && /(iphone|ipad|ipod|ios)/i.test(UA)
+export const isIos93 = isIos && UA.indexOf('os 9_3') > 0
```

Comment:
Only checking iOS 9.3 may not be future-proof if the same bug remains in 9.4 or future versions of iOS. We may need to do a regex match to check if the version number is greater than 9_3.